pages/api/getURI/[title].js

```diff
@@ -0,0 +1,33 @@
+import { getToken } from "next-auth/jwt";
+
+export default async function handler(req, res) {
```

Comment:
please use more descriptive names for your parameters, such as `request` and `response`

this will cause a naming conflict with line 15 though, but you will figure something out ;)